packages/jest-circus/src/utils.js

```diff
@@ -79,6 +79,7 @@ export const makeTest = (
     asyncError,
     duration: null,
     errors: [],
+    expired: false,
```

Comment:
I wonder if `timedOut ` would be better name?

Reply:
I was between both, not strongly opinionated either way

Reply:
more of a fan of `failureReason: 'timeout' | 'error' | 'assertion-error'` or something